### ISSUE TITLE
Downloading issue

### ISSUE THREAD

**Issue by panda-xinyi** [thunlp/CLEVER#2]
In CLEVER/src/Oscar /DOWNLOAD.md, both wget https://biglmdiag.blob.core.windows.net/oscar/datasets/$TASK_NAME.zip and wget https://biglmdiag.blob.core.windows.net/oscar/pretrained_models/$MODEL_NAME.zip are not accessible. Will they affect the testing result of CLEVER?

**Comment by yiranyyu** [comment]
No. These files are not used during our training or testing process.
